docs/code/IDataViewTypeSystem.md

```diff
@@ -661,154 +605,99 @@ This chapter contains information on the C# classes used to represent column
+exposed by the static `DateTimeDataViewType.Instance` property.
 
-The `DateTimeType` sealed class derives from `PrimitiveType` and is a
-singleton-class for the standard datetime type. The instance is exposed by the
-static `DateTimeType.Instance` property.
+### `DateTimeOffsetDataViewType` Sealed Class
 
-### `DateTimeZoneType` Sealed Class
+The `DateTimeOffsetDataViewType` sealed class derives from
+`PrimitiveDataViewType` and is a singleton-class for the standard datetime
+timezone type. The instance is exposed by the static
+`DateTimeOffsetDataViewType.Instance` property.
 
-The `DateTimeZoneType` sealed class derives from `PrimitiveType` and is a
-singleton-class for the standard datetime timezone type. The instance is
-exposed by the static `DateTimeType.Instance` property.
+### `TimeSpanDataViewType` Sealed Class
 
-### `TimeSpanType` Sealed Class
+The `TimeSpanDataViewType` sealed class derives from `PrimitiveDataViewType`
+and is a singleton-class for the standard datetime timezone type. The instance
+is exposed by the static `TimeSpanDataViewType.Instance` property.
 
-The `TimeSpanType` sealed class derives from `PrimitiveType` and is a
-singleton-class for the standard datetime timezone type. The instance is
-exposed by the static `TimeSpanType.Instance` property.
+### `KeyDataViewType` Sealed Class
 
-### `KeyType` Sealed Class
-
-The `KeyType` sealed class derives from `PrimitiveType` and instances
+The `KeyDataViewType` sealed class derives from `PrimitiveType` and instances
 represent key types.
 
-Notes:
-
-* Two key types are considered equal iff their kind, min, count, and
-  contiguous values are the same.
-
-* The static `IsValidDataKind` method returns true iff kind is `U1`, `U2`,
-  `U4`, or `U8`. These are the only valid underlying data kinds for key types.
-
-* The inherited `KeyCount` property returns the same value as the `Count`
-  property.
+Note that two key types are considered equal iff their kind and count are the
+same, though they may be different instances. (Unlike many of the
+aforementioned types, which are based on singleton instances.)
 
 ### `VectorType` Sealed Class
 
-The `VectorType` sealed class derives from `ColumnType` and instances
+The `VectorType` sealed class derives from `DataViewType` and instances
 represent vector types. The item type is specified as the first parameter to
 each constructor and the dimension information is inferred from the additional
 parameters.
 
-* The `DimCount` property indicates the number of dimensions and the `GetDim`
-  method returns a particular dimension value. All dimension values are non-
-  negative integers. A dimension value of zero indicates unknown (or variable)
-  in that dimension.
+* Thd `Dimensions` property is an immutable array of `int` indicating the
+  dimensions of the vector. This will be of length at least one. All dimension
+  values are non- negative integers. A dimension value of zero indicates
```

Comment:
`non- negative` extra space.

Reply:
I found a couple like these searching for `- `... probably some bad line breaking.